### PR TITLE
ButtonCascader: updates story from knobs to controls

### PR DESCRIPTION
**Pull request by thisisobate**
**What this PR does / why we need it**:
This PR updates story from knobs to controls
**Which issue(s) this PR fixes**:

Relates to #30182

### CODE REVIEW

File: packages/grafana-ui/src/components/ButtonCascader/ButtonCascader.story.tsx [hunk 1]
```diff
@@ -24,24 +28,24 @@ const getKnobs = () => {
...
 
-export const simple = () => {
-  const { disabled, text, options } = getKnobs();
-  return (
-    <ButtonCascader disabled={disabled} options={options} value={['A']}>
-      {text}
-    </ButtonCascader>
-  );
+const Template: Story<ButtonCascaderProps> = (args) => {
+  return <ButtonCascader {...args}>{args.children}</ButtonCascader>;
```
Comment (jackw) [inline packages/grafana-ui/src/components/ButtonCascader/ButtonCascader.story.tsx:43]: nit: we probably shouldn't spread `args.children` onto the ButtonCascader. We can separate the two by doing something like:

```
const Template: Story<ButtonCascaderProps> = ({ children, ...args }) => {
  return <ButtonCascader {...args}>{children}</ButtonCascader>;
};
```

File: packages/grafana-ui/src/components/ButtonCascader/ButtonCascader.story.tsx [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
-
-const getKnobs = () => {
-  return {
-    disabled: boolean('Disabled', false),
-    text: text('Button Text', 'Click me!'),
-    icon: select('Icon', ['plus', 'minus', 'table'], 'plus'),
-    options: object('Options', [
+  decorators: [withCenteredStory],
+  parameters: {
+    docs: {},
```
Comment (jackw) [inline packages/grafana-ui/src/components/ButtonCascader/ButtonCascader.story.tsx:13]: Is this empty object needed?

**Review by jackw — APPROVED** [review]
LGTM! 🚀 

Have left a couple of comments.

File: packages/grafana-ui/src/components/ButtonCascader/ButtonCascader.story.tsx (hunk 2 above)
Comment (thisisobate) [reply packages/grafana-ui/src/components/ButtonCascader/ButtonCascader.story.tsx:13]: I don't think so.
Would remove it right away.

Thanks for bringing this to my atention @jackw ! 🙌

**Pull request merged**